sippy-ng/src/components/Sidebar.js

```diff
@@ -82,6 +89,26 @@ export default function Sidebar(props) {
                   </ListItemIcon>
                   <ListItemText primary="Overview" />
                 </ListItem>
+                <CapabilitiesContext.Consumer>
+                  {(value) => {
+                    if (value.includes('releases')) {
+                      return (
+                        <ListItem
+                          key={'release-tags-' + index}
+                          component={Link}
+                          to={`/release/${release}/tags`}
+                          button
+                          className={classes.nested}
+                        >
+                          <ListItemIcon>
+                            <FileCopyOutlined />
+                          </ListItemIcon>
+                          <ListItemText primary="Payloads" />
+                        </ListItem>
+                      )
+                    }
+                  }}
+                </CapabilitiesContext.Consumer>
```

Comment:
Yea, React contexts are neat, I should've used them more for global data like the release instead of wiring it into props over and over and over again